build.py

```diff
@@ -165,4 +173,4 @@
     with open("../codal.json", 'w') as codal_json:
         json.dump(config, codal_json, indent=4)
 
-    build(True, True)
+    build(True, True)
```

Comment:
We can revert this change to reduce the diff.

I guess ending the file in a new line is a very old practice that probably doesn't matter that much these days, but I guess some habits are hard to break, and many projects still do this consistently.

Reply:
52806a5 sorts this. Gonna need a hell of a lot of rebasing later 😂